.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
I think prettier + format on save is enough to keep the code quality good. Of course we can also include eslint, but in my opinion it's too overkill